Public/New-PolarisStaticRoute.ps1

```diff
@@ -34,12 +34,15 @@ function New-PolarisStaticRoute {
         [Parameter( Mandatory = $True )]
         [string]
         $FolderPath,
```

Comment:
Could this be defaulted to "./", maybe? Then it doesn't need to be mandatory and we can just run:

`New-PolarisStaticRoute; Start-Polaris`

super simple :)